rovilay/phone_number-generator
# Ch write tests

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?    
    *write unit tests for endpoints

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/phone_number-generator.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ch-write-tests`
- set up the `.env` with the `.env.sample` file
- install dependencies - `yarn install`
- start the app - `yarn run test`
- all tests should pass

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
